File: proj/WinStore_FamilyAccounting/WinStore_FamilyAccountingCore/MasterData/CustomerMasterData.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinStore_FamilyAccountingCore.MasterData
{
    public class CustomerMasterData : MasterDataBase
    {
        public static readonly String FILE_NAME = "customer.xml";

        /// <summary>
        /// Construct
        /// </summary>
        /// <param name="coreDriver"></param>
        /// <param name="management"></param>
        /// <param name="id"></param>
        /// <param name="descp"></param>
        /// <exception cref="ArgumentNullException">argument is null</exception>
        public CustomerMasterData(CoreDriver coreDriver, MasterDataManagement management, MasterDataIdentity id,
                String descp)
            : base(coreDriver, management, id, descp)
        {
        }
    }
}

File: proj/CoreLibrary/src/com/jasonzqshen/familyaccounting/core/masterdata/CustomerMasterData.java
package com.jasonzqshen.familyaccounting.core.masterdata;

import com.jasonzqshen.familyaccounting.core.CoreDriver;
import com.jasonzqshen.familyaccounting.core.exception.NullValueNotAcceptable;

public class CustomerMasterData extends MasterDataBase {
	public static final String FILE_NAME = "customer.xml";

	/**
	 * 
	 * @param id
	 * @param descp
	 * @param parser
	 * @throws NullValueNotAcceptable 
	 */
	public CustomerMasterData(CoreDriver coreDriver, MasterDataManagement management, MasterDataIdentity id,
			String descp) throws NullValueNotAcceptable {
		super(coreDriver, management, id, descp);
	}
}
